### PR TITLE
Update themes.html

### PR DESCRIPTION
**Pull request by a1b2o3d46**

### CODE REVIEW

File: themes.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
             a.async = 1;
             a.src = g;
             m.parentNode.insertBefore(a, m)
-        })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
```
Comment (a1b2o3d46) [inline themes.html:36]: %appdata%\betterdiscord\themes\

**Comment by 0mniscient** [comment]
what?

**Comment by Serkan-devel** [comment]
ye

**Comment by 0mniscient** [comment]
what's this supposed to even do ? just removes my analytics

**Comment by Serkan-devel** [comment]
Why not add doubleclick.net advertising?

**Pull request closed without merging**